src/analytics/private/sqls/redshift/sp-migrate-session-to-v2.sql

```diff
@@ -0,0 +1,172 @@
+CREATE OR REPLACE PROCEDURE {{schema}}.sp_migrate_session_to_v2() 
+NONATOMIC 
+LANGUAGE plpgsql 
+AS $$ 
+
+DECLARE 
+
+record_number INT;
+
+log_name VARCHAR(50) := 'sp_migrate_session_to_v2';
+
+BEGIN 
+
+DROP TABLE IF EXISTS tmp_traffic_session_web;
+
+DROP TABLE IF EXISTS tmp_traffic_session_mobile;
+
+INSERT INTO
+    {{schema}}.session(
+        user_pseudo_id,
+        session_id,
+        user_id,
+        session_number,
+        session_start_time_msec,
+        event_timestamp,
+        process_info
+    )
+SELECT
+    user_pseudo_id,
+    session_id,
+    MAX(user_id),
+    MAX(session_number),
+    MIN(session_start_time_msec::bigint),
+    MAX(event_timestamp),
+    object('backfill_session', true)
+FROM
+    {{schema}}.event_v2 e
```

Comment:
It is better to make sure session_id is not null

and session_id is not null

Reply:
added